### PR TITLE
feat: new branch for chaos mesh release-2.4

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <im@strrl.dev>

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL @wuhuizuo

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by wuhuizuo — APPROVED** [review]
/lgtm

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -1138,6 +1138,7 @@ ti-community-label:
       - "needs-cherry-pick-2.1"
       - "needs-cherry-pick-2.2"
       - "needs-cherry-pick-2.3"
+      - "needs-cherry-pick-2.4"
```
[the remark this answers is not in the document]
Comment (STRRL) [reply prow/config/external_plugins_config.yaml:1141]: no, I have manually append this label.

**Pull request merged**